src/main/java/hudson/remoting/Capability.java

```diff
@@ -145,6 +146,7 @@ protected void annotateClass(Class<?> c) throws IOException {
     /**
      * The opposite operation of {@link #writePreamble(OutputStream)}.
      */
+    @SuppressFBWarnings(value = "OBJECT_DESERIALIZATION", justification = "Capability is used for negotiating channel between authorized agent and server.")
```

Comment:
Perhaps you can use a justification that the ObjectInputStream used in this method clearly uses whitelisting and proper deserialization hygiene.